adapters/InMobi/InMobiAdapter/GADMediationAdapterInMobi.h

```diff
@@ -11,21 +11,24 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//
 
 #import <Foundation/Foundation.h>
 #import <GoogleMobileAds/GoogleMobileAds.h>
 #import <InMobiSDK/InMobiSDK.h>
 
 typedef NS_ENUM(NSInteger, GADMAdapterInMobiErrorCode) {
-  /// Missing server parameters.
-  GADMAdapterInMobiErrorInvalidServerParameters = 101,
-  /// Banner Size Mismatch.
-  GADMAdapterInMobiErrorBannerSizeMismatch = 102,
-  /// An ad is already loaded for this network configuration.
-  GADMAdapterInMobiErrorAdAlreadyLoaded = 103,
-  /// InMobi native ad returned with missing native assets, or required image assets failed to
-  /// download.
-  GADMAdapterInMobiErrorMissingNativeAssets = 104
+    /// Missing server parameters.
+    GADMAdapterInMobiErrorInvalidServerParameters = 101,
+    /// Banner Size Mismatch.
+    GADMAdapterInMobiErrorBannerSizeMismatch = 102,
+    /// An ad is already loaded for this network configuration.
+    GADMAdapterInMobiErrorAdAlreadyLoaded = 103,
+    /// InMobi native ad returned with missing native assets, or required image assets failed to
+    /// download.
+    GADMAdapterInMobiErrorMissingNativeAssets = 104,
+    /// An ad is not ready.
+    GADMAdapterInMobiErrorAdNotReady = 105
```

Comment:
Can you clarify what does "not ready" mean here?

Reply:
its means not ready to shown i.e still loading